SETUP.md

```diff
@@ -97,11 +97,10 @@ The following command automates this substitution.
 
 ```sh
 # Edit ci/deploy.sh with travis secure env variables generated by travis encrypt-file
-TRAVIS_ENCRYPT_ID=`grep \
-  --only-matching --perl-regexp \
-  --regexp='(?<=encrypted_)[a-zA-Z0-9]+(?=_key)' \
-  travis-encrypt-file.log`
-sed --in-place "s/f2f00aaf6402/$TRAVIS_ENCRYPT_ID/g" deploy.sh
+TRAVIS_ENCRYPT_ID=`python -c "import re; \
+                  text = open('travis-encrypt-file.log').read(); \
+                  print(re.search('encrypted_([a-f0-9]+)_key', text).group(1))"`
```

Comment:
Our style for the shell commands is 2 space indent

Reply:
Okay will update accordingly